.github/workflows/knative-boilerplate.yaml

```diff
@@ -85,7 +85,7 @@ jobs:
           boilerplate-check check \
             --boilerplate ./hack/boilerplate/boilerplate.${{ matrix.extension }}.txt \
             --file-extension ${{ matrix.extension }} \
-            --exclude "(vendor|third_party)/" |
+            --exclude "(vendor|third_party|control-plane/pkg/contract)/" |
```

Comment:
Knobots will override this change: https://github.com/knative-sandbox/eventing-kafka-broker/blob/a35228912d21476c12dd42de228e5c1036015afa/.github/workflows/knative-boilerplate.yaml#L15-L16

Reply:
Crap!

I would like to still merge this change, and talk to productivity WG about this problem.

I spent too much time on this PR because of these kind of logistics stuff.